CapacitorGoogleFit.podspec

```diff
@@ -0,0 +1,17 @@
+require 'json'
```

Comment:
Is there a point to add a podspec file in a plugin that does only work on Android?

Reply:
This file was also auto-generated i removed it.